### PR TITLE
✨ 💥 Feature/webhook verification

### LINKED ISSUE #110: Add function to verify specifically the webhook's HMAC
Following up on discussion #105 we should have a simple function to verify the webhook and it should be able to just call existing code.

```python
# Simple but a great building block
# https://shopify.dev/apps/webhooks/configuration/https#step-5-verify-the-webhook -> click python
def verify_webhook(data: str, hmac_header: str, api_secret_key: str) -> bool:
    ...
```

### PR DESCRIPTION
**Pull request by lishanl**
fix #110 

- rename hmac functions param `is_base64` to `use_base64`
- add ~~`verify_webhook`~~ `is_webhook_valid` function
- add tests

### CODE REVIEW

File: spylib/token.py [hunk 1]
```diff
@@ -400,3 +401,11 @@ def load(cls, store_name: str):
         therefore the developer should override this.
         """
         pass
+
+
+def verify_webhook(data: str, hmac_header: str, api_secret_key: str) -> bool:
```
Comment (hillairet) [inline spylib/token.py]: If you return a boolean, I think a better name would be this:
```suggestion
def is_webhook_valid(data: str, hmac_header: str, api_secret_key: str) -> bool:
```

This way one can have a readable if statement:
```python
if is_webhook_valid():
```

**Review by hillairet — CHANGES_REQUESTED** [review]
Because you changed the arguments of a couple of functions, you should probably use a breaking change emoji for the PR title.

**Review by hillairet — APPROVED** [review]
Cool stuff!

**Review by qw-in — APPROVED** [review]
Awesome thanks @lishanl! Excited to rip out our custom webhook code

**Pull request merged**